capstone/src/main/java/com/step902020/capstone/Individual.java

```diff
@@ -85,4 +81,32 @@ public String getUserType() {
   public String getImage() {
     return image;
   }
+
+  public Set<String> getSavedEvents() {
+    return savedEvents;
+  }
+
+  public void editFirstName(String firstName) {
+    this.firstName = firstName;
+  }
+
+  public void editLastName(String lastName) {
+    this.lastName = lastName;
+  }
+
+  public void editUniversity(String university) {
```

Comment:
Should users be able to change their universities?

Reply:
You're right it's probably not a good idea.